app/components/wds-community-bar.js

```diff
@@ -0,0 +1,43 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+import track from '../utils/track';
+
+const discussionsTitleKey = 'community-header-discuss';
+
+export default Component.extend({
+	classNames: ['wds-community-bar'],
+
+	// filter out discussion link from navigation model
+	navigation: computed('model', function () {
+		return this.model.navigation.filter((navigationItem) => {
+			return navigationItem.title.key !== discussionsTitleKey;
+		});
+	}),
+
+	// discussions link is a separate item in community bar
+	discussionsLink: computed('model', function () {
+		const discussionsLink = this.model.navigation.filter((navigationItem) => {
```

Comment:
to not do this filtering twice - one for navigation and one for discussionsLink
maybe it would make sense to do it on init?